packages/serve/src/index.ts

```diff
@@ -1,27 +1,27 @@
 import { devServerOptionsType } from "./types";
 
+const WEBPACK_PACKAGE = process.env.WEBPACK_PACKAGE || "webpack";
+const WEBPACK_DEV_SERVER_PACKAGE = process.env.WEBPACK_DEV_SERVER_PACKAGE || "webpack-dev-server";
```

Comment:
@snitin315 We should document `WEBPACK_DEV_SERVER_PACKAGE` after merge this

Reply:
I will take care of it 👍🏻